enforcer/datapath.go

```diff
@@ -191,7 +191,7 @@ func NewWithDefaults(
 // Enforce implements the Enforce interface method and configures the data path for a new PU
 func (d *Datapath) Enforce(contextID string, puInfo *policy.PUInfo) error {
 
-	puContext, err := d.contextTracker.Get(contextID)
+	puContext, err := d.contextTracker.GetReset(contextID)
```

Comment:
I am not sure we should expire the context tracker .. this is explicitly done by enforce/unenforce